packages/storybook/stories/va-progress-bar.stories.jsx

```diff
@@ -17,8 +17,8 @@ const Template = ({ 'enable-analytics': enableAnalytics, label, percent }) => (
 
 const defaultArgs = {
   'enable-analytics': false,
-  'label': 'Add a label here',
-  'percent': 35,
+  'label': undefined,
+  'percent': 48.123456,
```

Comment:
I made these changes to make it easier for Courtney to test accessibility. I can revert them before merging this PR if requested.